tmail-backend/apps/distributed/src/test/java/com/linagora/tmail/james/app/CombinedIdentityJamesServerTest.java

```diff
@@ -84,4 +85,12 @@ void shouldUseCombinedUsersRepositoryWhenSpecified(GuiceJamesServer server) {
         assertThat(server.getProbe(UsersRepositoryClassProbe.class).getUserRepositoryClass())
             .isEqualTo(CombinedUsersRepository.class);
     }
+
+    @Test
+    void shouldAllowUserSynchronisation(GuiceJamesServer server) throws Exception {
+        server.getProbe(DataProbeImpl.class)
+            .fluent()
+            .addDomain("james.org")
+            .addUser("james-user@james.org", "123456");
```

Comment:
Is adding a user through the probe the test here?

Reply:
Yes